application/account-management/Api/Api.csproj

```diff
@@ -6,11 +6,16 @@
         <RootNamespace>PlatformPlatform.AccountManagement.Api</RootNamespace>
         <Nullable>enable</Nullable>
         <ImplicitUsings>enable</ImplicitUsings>
+        <ContainerFamily>jammy-chiseled</ContainerFamily>
         <OpenApiDocumentsDirectory>$(MSBuildProjectDirectory)/../WebApp/lib/api/</OpenApiDocumentsDirectory>
         <OpenApiGenerateDocuments>true</OpenApiGenerateDocuments>
         <OpenApiGenerateDocumentsOnBuild>true</OpenApiGenerateDocumentsOnBuild>
     </PropertyGroup>
 
+    <ItemGroup>
+        <ContainerPort Include="8443" Type="tcp" />
```

Comment:
I would like to change this to the default port `8080`, which means we can delete the code setting the port in Shared-kernel. See: https://github.com/platformplatform/PlatformPlatform/commit/c4cb25c514ad85c35c54b712a2e8f11f600f21d7.

While it's the default port I think it's nice to have it explicit.

Also this should be moved up just belo the `<ContainerFamily>jammy-chiseled</ContainerFamily>`

Reply:
Ok I can see that the 8080 became the default port for .NET 8. We can also configure this by setting one the following environment variables ASPNETCORE_HTTP_PORTS, ASPNETCORE_HTTPS_PORTS, and ASPNETCORE_URLS. Do you have a preference?
I would prefer not to have at all in this case.